<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>717</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>533</width>
    <height>546</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabBathymetry">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generate">
      <attribute name="title">
       <string>Input Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>175</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Grid:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTreeWidget" name="bat_gridTree">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <column>
               <property name="text">
                <string>Filename</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Filepath</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="bat_removeGrdBtn">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(204, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="bat_gridBox"/>
              </item>
              <item>
               <widget class="QPushButton" name="bat_loadGrdBtn">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(78, 154, 6)</string>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>175</height>
          </size>
         </property>
         <property name="title">
          <string>XYZ data:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTreeWidget" name="bat_xyzTree">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <column>
               <property name="text">
                <string>Filename</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Filepath</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="bat_removeXYZBtn">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(204, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="bat_XYZBox"/>
              </item>
              <item>
               <widget class="QPushButton" name="bat_loadXYZBtn">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(78, 154, 6);</string>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>175</height>
          </size>
         </property>
         <property name="title">
          <string>Land:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTreeWidget" name="bat_landTree">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>80</height>
               </size>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <column>
               <property name="text">
                <string>Filename</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Filepath</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="bat_removeLandBtn">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(204, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="bat_landBox"/>
              </item>
              <item>
               <widget class="QPushButton" name="bat_loadLandBtn">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(78, 154, 6);</string>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bat_inputLabel_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="bat_outputPath"/>
         </item>
         <item>
          <widget class="QPushButton" name="bat_fsOutput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>66</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Run</string>
      </attribute>
      <widget class="QPushButton" name="bat_generateBtn">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Load">
      <attribute name="title">
       <string>View and Edit</string>
      </attribute>
      <widget class="QLabel" name="bat_inputLabel_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>117</width>
         <height>17</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Load bathymetry</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>153</y>
         <width>251</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="bat_batPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="bat_loadBatBtn">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="bat_inputLabel_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save bathymetry</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bat_saveBatBtn">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
